### ISSUE TITLE
Web accessibility: Dashboard 

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/libsys-airflow#634]
There are a few basic accessibility issues to address on the dashboard page:

- [x] set `role='heading'` here https://github.com/sul-dlss/libsys-airflow/blob/main/libsys_airflow/plugins/vendor_app/templates/vendors/dashboard.html#L6
- [x] Structure headings. The page heading is an `h1` element, then "In Progress" and "Errors" are `h3`, however they should follow the structure and be `h2` elements, or the page heading should be shifted to an `h2`.
- [x] All of the text areas (titles, tab headings, etc) need to be included in an aria landmark: https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/region.html
- [x] Tab headings need appropriate role organization (https://www.w3.org/WAI/standards-guidelines/act/rules/bc4a75/proposed/) - this seems tenuous, the SiteImprove message is "Make sure required-owned elements are nested correctly under the context-giving element.
- [x] Ensure color contrasts are at least 7 (per SiteImprove)

Note: It may be easiest to understand the above points if you install the SiteImprove extension (https://www.siteimprove.com/integrations/browser-extensions) in order to see the specifics (especially on the color contrast)
